mcdc/Earthfile

```diff
@@ -0,0 +1,30 @@
+VERSION 0.6
+
+mcdc:
+  FROM registry.gitlab.com/gtd-gmbh/mcdc-checker/mcdc-checker
+  #COPY mcdc_checker-patched.py /app/src/mcdc_checker.py
+  COPY mcdc_checker.patch /tmp/mcdc_checker.patch
+  RUN pacman -Sy --noconfirm patch
+  RUN patch /app/src/mcdc_checker.py < /tmp/mcdc_checker.patch
+  # Clean up the modified patch file and pacman cache.
+  RUN pacman -Rsscn --noconfirm patch && pacman -Scc --noconfirm
+
+package-list:
+  FROM ../spaceros+workspace
+  RUN colcon list | grep 'cmake)' | awk '{print $2}' > workspace-packages.txt
```

Comment:
Fetch every package that is ament_cmake or cmake (and thus is likely) to contain C/C++ artifacts.